Automattic/woocommerce-services
# Update TaxJar proxy to full integration

This PR seeks to:

* Remove the dependence on a separate TaxJar plugin
* Only integrate sales tax calculation - order syncing and nexus functionality is not required

To test:
* Ensure your store is in a supported country ([#](https://developers.taxjar.com/api/reference/#countries))
* Go to WooCommerce > Settings > Tax and enable "Automated taxes"
* Verify that any previously configured tax rates have been exported to CSV (uploads dir)
* Verify that taxes are calculated in cart (behavior will vary based on store location)
* Verify that taxes are calculated for manually created orders in the dashboard

cc: @allendav
